### DISCUSSION TITLE
简悦适配了一些常见的科研期刊类网站

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2103] [站点适配]
## 描述

> 简悦适配了一些常见的科研期刊类网站。

## 特点

> 简悦适配的这些科研期刊，并不是 **简单的将正文提取出来**，而是 **根据学术文章一般有固定的格式**，如：`title` `abstract` `results` `discussions` `methods` `references` `acknowledgements` `highlights` 进行适配的。

所以，满足一般科研期刊的使用规律，且可显示 **数学公式 LateX** 等仅 **简悦的阅读模式独有的功能**。

## 如何更新

> 请看 [官方适配源](http://ksria.com/simpread/docs/#/站点适配源?id=官方适配源)，更新后请打开下面的一些示例网站查看。

## 包括

- [x] [知网](https://github.com/Kenshin/simpread/discussions/3537)

   > 知网是我从开发简悦以来适配最困难的一个网站，不是说它有多难适配，而是我无法进入，所以辗转了很多方式，得到了很多同学们以及简悦用户的支持，才能完成这个插件。
   > 感谢那些为此付出时间和心血的简悦用户，特意说下简悦用户 [@ilovepaper](https://github.com/Kenshin/simpread/discussions/2103#discussioncomment-2349356)，一共为此：18次回复与15封邮件，感谢付出 👏 👏

- [x] https://www.sciencedirect.com/

  > 示例 https://www.sciencedirect.com/science/article/pii/S095965262033136X https://www.sciencedirect.com/science/article/pii/S2590238520303775 https://www.sciencedirect.com/science/article/pii/S0022169405002726

- [x] https://www.ncbi.nlm.nih.gov/

  > 示例 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5732142/ https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3471287/

- [x] https://journals.biologists.com/

  > 示例 https://journals.biologists.com/bio/article/2/12/1397/34350/Evaluation-of-visible-implant-elastomer-tags-in https://journals.biologists.com/bio/article/10/4/bio058602/261692/Visualization-of-X-chromosome-reactivation-in

- [x] https://www.cell.com/cell/home

  > 示例 https://www.cell.com/cell/fulltext/S0092-8674(21)00438-4 https://www.cell.com/cell/fulltext/S0092-8674(21)00439-6 https://www.cell.com/current-biology/fulltext/S0960-9822(20)30360-2?utm_medium=homepage https://www.cell.com/neuron/fulltext/S0896-6273(20)30186-0#secsectitle0025

- [x] https://elifesciences.org/

  > 示例 https://elifesciences.org/articles/62585  https://elifesciences.org/articles/43140 https://elifesciences.org/articles/39945

- [x] https://www.pnas.org/

  > 示例 https://www.pnas.org/content/117/11/5547  https://www.pnas.org/content/117/11/5726

- [x] https://science.sciencemag.org/

  > 示例 https://science.sciencemag.org/content/367/6484/eaay6690.full https://science.sciencemag.org/content/368/6489/409 https://science.sciencemag.org/content/372/6541/twil https://science.sciencemag.org/content/367/6484/eaay6690.full

- [x] https://www.nature.com/

  > 示例 https://www.nature.com/articles/s41586-019-1563-y https://www.nature.com/articles/s41423-020-0402-2 https://www.nature.com/articles/s41585-020-0312-1 https://www.nature.com/articles/s41467-020-15272-2

- [x] https://www.frontiersin.org/journals

  > 示例 https://www.frontiersin.org/articles/10.3389/fnbeh.2021.658686/full https://www.frontiersin.org/articles/10.3389/fnbeh.2021.658037/full https://www.frontiersin.org/articles/10.3389/fnbeh.2021.630889/full https://www.frontiersin.org/articles/10.3389/fnbeh.2014.00121/full

## 提交

> 如果上述期刊并没有你需要的，请在此留言告诉我。

## 已知问题

- 简悦根据 URL 进行识别，所以一旦 URL 的结构改变会造成无法识别的问题，期刊类的页面在 URL 上相对更加的复杂，所以有可能会出现个别页面无法识别的情况。（如发生的话，请在这里告知与我~

- 阅读模式只是将正文 / 图片 资源提取出来，除此以外的内容，如：对视频的兼容性有一定的匹配度，所以不是每个包含视频资源的页面都会被良好解析。（上述我测试过的页面是没问题的）

- 简悦尊重任何形式的知识产权，所以 **针对一些「付费墙」页面，简悦没有适配它们**。

## 引申

一般情况下，科研类工作者可能会用到维基百科，建议看下关于维基百科的一些 **[重要提示](https://github.com/Kenshin/simpread/discussions/2347)**

**Comment by leeeeeeeee2** [comment]
- [ ] 美国气象学会期刊，付费文章界面https://journals.ametsoc.org/view/journals/apme/60/4/JAMC-D-20-0209.1.xml
- [ ] 同上，免费文章界面https://journals.ametsoc.org/view/journals/apme/60/4/JAMC-D-20-0252.1.xml
谢谢作者！

**Reply by Kenshin** [reply]
嗨，可以的。
不过，付费页面是不会被适配的（因为我无法查看，因此付费与免费的界面结构上是不一样的。

**Reply by leeeeeeeee2** [reply]
谢谢！😃

**Reply by leeeeeeeee2** [reply]
作者你好，发现这个网页的公式没有被很好的显示https://www.nature.com/articles/s41597-020-00654-4
还请关注下！谢谢你的作品！
公式位置：Sensitivity analysis部分的上方两行之前

**Reply by Kenshin** [reply]
嗨，其实是可以显示的，如下图

![image](https://user-images.githubusercontent.com/81074/118425545-f9dd2b80-b6fb-11eb-8e77-0da341d36ac0.png)

***

> 解决方案

1. 等页面加载完毕后，手动拉到最底部
2. 将这个页面加入到 [延迟加载列表](http://ksria.com/simpread/docs/#/%E8%AF%8D%E6%B3%95%E5%88%86%E6%9E%90%E5%BC%95%E6%93%8E?id=%e5%bb%b6%e8%bf%9f%e5%8a%a0%e8%bd%bd)

PS：如果仍未解决的话，应该是 2.1.0 版的问题，截图是在新版截取的，发布后自然就可以了（即将发布）

**Comment by PayneSnow** [comment]
感谢。看了一下列表，基本上都是理工科的期刊。希望适配一下偏向人文社科的数据库。期待适配一下 以下三个数据库。里面的不同期刊的格式和版面设计基本上是一致的。里面有的文章是免费全文查看的，有的是付费的。
- 泰勒·佛朗西斯数据库 https://www.tandfonline.com/
- 例子：https://www.tandfonline.com/doi/full/10.1080/19392397.2013.791041
- sage数据库
- 例子：https://journals.sagepub.com/doi/full/10.1177/0731121417719699
- wiley数据库
- 例子：https://anthrosource.onlinelibrary.wiley.com/doi/full/10.1111/aman.13558

**Reply by Kenshin** [reply]
@PayneSnow 

收到，近期在忙新版的事情，等发布后会再适配它们的。
也谢谢帮忙提交~ 🙏

**Reply by PayneSnow** [reply]
> @PayneSnow
> 
> 收到，近期在忙新版的事情，等发布后会再适配它们的。
> 也谢谢帮忙提交~ 🙏

感谢。新版看到了，太酷炫了。继续走在封神的道路上。进化速度太快了。

**Reply by PayneSnow** [reply]
> @PayneSnow
> 
> 收到，近期在忙新版的事情，等发布后会再适配它们的。
> 也谢谢帮忙提交~ 🙏

辛苦了。坐等移动端了。

**Reply by Kenshin** [reply]
> > @PayneSnow
> > 收到，近期在忙新版的事情，等发布后会再适配它们的。
> > 也谢谢帮忙提交~ 🙏
> 
> 感谢。新版看到了，太酷炫了。继续走在封神的道路上。进化速度太快了。

哈哈，封神不敢当，希望你们能喜欢就好（ 2.2.0 发布也引起了一些小麻烦，好在都可以解决。

**Comment by chenxinfeng4** [comment]
太感谢了。不过，在已有的“适配”里面，还是有欠缺
## 1. Abstract 和 正文丢失
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5732142/

## 2. 换行错误。
https://www.cell.com/neuron/fulltext/S0896-6273(20)30186-0  

## 3. Abstract 被折叠
https://www.frontiersin.org/articles/10.3389/fnbeh.2021.658037/full

**Reply by Kenshin** [reply]
嗨，收到，因为这次 **Webstore 审核的超级快**，导致 2.2.0 突然发布了，导致这几天有点手忙脚乱。

关于你提到的问题，我会过阵子逐一解决。（你的邮件我也已回复。

**Reply by chenxinfeng4** [reply]
多谢，太有效率了

**Comment by aifact** [comment]
https://link.springer.com/article/10.1007/s00425-004-1332-4#citeas
标题太长，阅读模式下，标题的最后一行被遮盖
![image](https://user-images.githubusercontent.com/79238945/118929201-cd811380-b976-11eb-9f92-074e25b06a94.png)

**Reply by Kenshin** [reply]
收到~

**Comment by fornewfork** [comment]
请求适配这几个网站，谢谢！

- 牛津大学出版社数据库：https://academic.oup.com
例：https://academic.oup.com/imajna/article-abstract/41/2/935/5877757?redirectedFrom=fulltext

- 美国数学学会数据库：https://www.ams.org
例：https://www.ams.org/journals/mcom/2021-90-329/S0025-5718-2021-03604-2/home.html

- IOPscience数据库：https://iopscience.iop.org/
例：https://iopscience.iop.org/article/10.1088/1361-6420/abf36c

- 美国工业和应用数学学会数据库：https://epubs.siam.org
例：https://epubs.siam.org/doi/abs/10.1137/19M1294216

**Reply by Kenshin** [reply]
嗨，没问题，不过得等过一小段时间了，目前在忙 2.2.0 相关的事情。 😅

**Reply by hujie12306** [reply]
RCS publish: https://pubs.rsc.org
例： https://pubs.rsc.org/en/content/articlelanding/2021/an/d1an01801a

**Comment by Kenshin** [comment]
再补一个 https://github.com/Kenshin/simpread/issues/2596

**Comment by Kenshin** [comment]
再补一个 https://github.com/Kenshin/simpread/issues/3063

RCS publish: https://pubs.rsc.org
例： https://pubs.rsc.org/en/content/articlelanding/2021/an/d1an01801a

**Comment by MengjiaHe** [comment]
https://pubs.acs.org/

**Reply by Kenshin** [reply]
收到~

**Comment by Kenshin** [comment]
> 看到这里的各位，[知网已经适配完成](https://github.com/Kenshin/simpread/discussions/3537)。感谢 @ilovepaper 的付出，一共为此：18次回复与15封邮件，感谢付出 👏 👏

***

您好~cnki论文的在线阅读界面，文本可以很好的提取，只是图片显示不出（往往是公式和表格），不知是否有好的解决办法。[链接](https://x.cnki.net/read/article/xmlonline?filename=SHJR202106006&tablename=CJFDTOTAL&dbcode=CJFD&topic=&fileSourceType=1&taskId=&from=ReadingHistory&groupid=&appId=CRSP_BASIC_PSMC&act=&ts=1647154961&customReading=)，辛苦您了！

via @ilovepaper

**Reply by Kenshin** [reply]
@ilovepaper

嗨，我帮你新起了一层，知网我是没法访问的。

简悦用户提供了一种方案，但我不知道你给出链接（的网页）与 https://github.com/Kenshin/simpread/issues/3470 有什么区别？

**Reply by ilovepaper** [reply]
> @ilovepaper
> 
> 嗨，我帮你新起了一层，知网是我没法访问的。
> 
> 简悦用户提供了一种方案，但我不知道你给出链接（的网页）与 #3470 有什么区别？

您好！非常开心这么快就收到了您的回复。感谢帮助另起一层。

可能我提供的这个[链接](https://x.cnki.net/read/article/xmlonline?filename=DOUB201202018&tablename=CJFDTOTAL&dbcode=CJFD&topic=&fileSourceType=1&taskId=&from=&groupid=&appId=CRSP_BASIC_PSMC&act=&ts=1647173275&customReading=)是凭账号密码不借助vpn也可阅读全文的（或许可以试读一部分），而#3470这位朋友用html阅读全文的方式是需要登录vpn的（论文内容一致，显示样式有一些差异）。

在我提供的链接中原网站打开样式如下。![Snipaste_2022-03-13_20-08-21](https://user-images.githubusercontent.com/57012991/158059539-9dfa7604-0af2-4a24-9eb1-d11827558b6e.png)使用简悦后，对于没有图表的论文，识别时能精准获取标题，但是会缺少正文前的作者信息、摘要和正文后的参考文献；对于有图表的论文，除了以上问题外，还无法显示图表![Snipaste_2022-03-13_20-14-11](https://user-images.githubusercontent.com/57012991/158059570-36e7cbf4-da77-4f0b-808b-bac37fc4a2fa.png)

在html里直接阅读论文，除了有无法识别标题的问题外，不知为何我的电脑上直接浏览论文时图表就无法显示了（切换了多个网络/浏览器都如此），就像是网速差刷不出图片的样子，因此简悦里也没有完整的图表了。（图中可以看到，前面一张图片可以正常显示，但后面一张则无法正常显示）
![image](https://user-images.githubusercontent.com/57012991/158059708-354dd856-cbc1-4313-a2f0-cbe958a5b034.png)

诉求：无论通过哪类链接，简悦后如果能够获取包含【标题+作者+摘要+正文（含所有图表）+参考文献】的全部内容就最好了^_^！谢谢您！ps：一句题外话，插件真的很好用！会强推给朋友们的。

**Reply by Kenshin** [reply]
okay 我会研究下，谢谢喜欢以及帮忙推荐简悦。🙏

只要能进入页面就方便些，但现在仍然有个「问题」：**适配规则依赖于 URL的一致（相似）性**，所以我需要你提供一些 URL 地址，我看看是否有共同点。

**Reply by ilovepaper** [reply]
> okay 我会研究下，谢谢喜欢以及帮忙推荐简悦。🙏
> 
> 只要能进入页面就方便些，但现在仍然有个「问题」：**适配规则依赖于 URL的一致（相似）性**，所以我需要你提供一些 URL 地址，我看看是否有共同点。

谢谢您的耐心~~除了之前的链接再补充两个链接给您参考。
[链接1](https://x.cnki.net/read/article/xmlonline?filename=GLSJ201307006&tablename=CJFDTOTAL&dbcode=CJFD&topic=&fileSourceType=1&taskId=&from=&groupid=&appId=CRSP_BASIC_PSMC) [链接2](https://x.cnki.net/read/article/xmlonline?filename=JRYJ200907013&tablename=CJFDTOTAL&dbcode=CJFD&topic=&fileSourceType=1&taskId=&from=&groupid=&appId=CRSP_BASIC_PSMC&act=&ts=1647230839&customReading=) 
另外我考虑了一下是否是延迟加载的问题，但加入延迟加载列表后依旧存在图片裂开的现象。
![image](https://user-images.githubusercontent.com/57012991/158114460-f38308ef-1365-496c-85e3-fa952d2b12f3.png)

期待问题的解决！好的软件当然值得推荐给更多朋友。谢谢您~~~~

**Reply by Kenshin** [reply]
Cool~

我想到了解决链接不同性的办法，另外，有没有带图片的论文，我找了半天都没有找到，有种大学时代论文答辩的既视感。 😂 

我需要：

1. 带图片的论文
2. 不需要给我看链接
3. 截图给我，包括这篇论文的标题

**Reply by ilovepaper** [reply]
> Cool~
> 
> 我想到了解决链接不同性的办法，另外，有没有带图片的论文，我找了半天都没有找到，有种大学时代论文答辩的既视感。 😂
> 
> 我需要：
> 
> 1. 带图片的论文
> 2. 不需要给我看链接
> 3. 截图给我，包括这篇论文的标题

哇！您的回复总是那么及时。刚才给您发的两篇链接是包含公式（图片）的论文，题目分别为：融资约束、现金持有与过度投资，机构投资者羊群行为与股价崩盘风险。截图如下：
![image](https://user-images.githubusercontent.com/57012991/158118745-41f4c4d2-1f03-40ee-ba24-fedabaf300a2.png)
![image](https://user-images.githubusercontent.com/57012991/158118755-87e8a67e-9ee5-4bdb-b27d-22109899b29a.png)
![image](https://user-images.githubusercontent.com/57012991/158118770-c9a24f86-5211-4a54-b285-33be5a91dc26.png)
![image](https://user-images.githubusercontent.com/57012991/158118777-54447393-3a23-4332-9c71-17523fab7125.png)
如果还有其他需要提供的信息再告诉我~~感恩！

**Reply by Kenshin** [reply]
哈哈，因为我目前正好在做这个事情，所以回复会比较快，看下效果

![image](https://user-images.githubusercontent.com/81074/158119827-09dfdd05-03c4-42b9-9e76-b25102f8cc34.png)

![image](https://user-images.githubusercontent.com/81074/158119881-d2c9f8cc-b299-44cb-b085-6a230bf6fcff.png)

**Reply by ilovepaper** [reply]
是的，想要的就是这个效果。太赞了！

**Reply by Kenshin** [reply]
https://simpread.ksria.cn/plugins/details/yHqalbdeNq

你可以先安装这个插件试试，不用理会插件的名称，下面是插件的简单介绍

![image](https://user-images.githubusercontent.com/81074/158128254-7bbc6289-0f84-4daa-8dbc-1fc641912ae3.png)

![image](https://user-images.githubusercontent.com/81074/158128402-ecb247bb-50c9-4f9a-9bf8-6a96bab34b54.png)

另外，注意事项：

1. 知网的 URL 没有规范性，所以暂时无法自动进入阅读模式，需要手动进入。
2. 我不确定我的界面 HTML 结构与你是否一致，所以你那边是否可以正常使用是否不得而知。
3. 如果你使用了同步助手，并且你的位置与图二不一致的话，请使用 https://github.com/Kenshin/simpread/discussions/2342 解决。

请帮忙多测试一些网页，如遇到问题，请（带标题）截图。

**Reply by ilovepaper** [reply]
您好！看到您如此效率地提供了解决方案真的十分惊喜！
我确实使用了同步助手，因此花了一些时间解决插件安装的问题。然后插件顺利出现了，但进入简悦模式的时候，出现下列两种提示：
![image](https://user-images.githubusercontent.com/57012991/158169125-3c1aac2c-9dec-4b0f-a109-5a8ea09d8be1.png)
测试链接依旧是刚才的链接1和链接2，截图也同上。简悦后页面是空白的。
辛苦您了！

**Reply by Kenshin** [reply]
emmm 可能我的 HTML 与你的不一致，请问你能使用 Telegram 吗？明天加你详细聊聊。

**Reply by ilovepaper** [reply]
> emmm 可能我的 HTML 与你的不一致，请问你能使用 Telegram 吗？明天加你详细聊聊。

能得到您的帮助就太好了！我没有Telegram，刚刚去了解了一下，或许我可以用GoogleVoice注册一个（但24小时内或许还无法进行私聊？）。不知道您是否方便通过qq、邮件或者其他联络方式沟通呢？谢谢！

**Reply by Kenshin** [reply]
> 能得到您的帮助就太好了！我没有Telegram，刚刚去了解了一下，或许我可以用GoogleVoice注册一个（但24小时内或许还无法进行私聊？）。不知道您是否方便通过qq、邮件或者其他联络方式沟通呢？谢谢！

哈哈，如果没有的话，不用那么麻烦。

不过，我让简悦其他用户（可以正常访问知网的用户）测试了这个插件都是可以使用的，所以你可以确认下是否真的无法使用。

比如：

- 韩国数字图书馆建设的经验和启示
- 先秦文学的翻译与研究——访法国汉学家雷米·马修
- 融资约束、现金持有与过度投资
- 先秦“以丑为美”文学审美观探析

所以，你可以试试这几个页面是否都不好使？

**Reply by ilovepaper** [reply]
您好！我刚才继续进行了尝试。
1、针对x.cnki.net/read/article域名下的链接进行尝试的结果：不开插件时图片获取出现问题，开了插件以后直接变成空白页或者句号（尝试用某些其他阅读模式插件下是可以获取图片的，只是并无简悦优秀的标注功能）
[链接参考](https://x.cnki.net/read/article/xmlonline?filename=JRYJ200907013&tablename=CJFDTOTAL&dbcode=CJFD&topic=&fileSourceType=1&taskId=&from=&groupid=&appId=CRSP_BASIC_PSMC&act=&ts=1647335817&customReading=#circle=on)
标题：融资约束、现金持有与过度投资

2、刚才对html进行阅读进行尝试是成功的！
进入渠道：
![image](https://user-images.githubusercontent.com/57012991/158347239-9e9ea8f4-ea71-4cb7-871e-8ab7ccf0092c.png)
插件失灵的原因和尝试：
1、chrome插件之间的冲突。于是我在隐私模式下仅打开简悦，但是依旧显示正文获取异常。
2、简悦内部插件之间冲突，比如分栏阅读等。禁用后问题依然存在。
不知您推断x.cnki.net/read/article域名无法识别可能还有什么原因呢？感谢~~~~！

**Reply by Kenshin** [reply]
1. 关闭同步助手
2. 禁用出这个插件以外的全部插件
3. 今天我更新了下插件的功能，所以重新安装这个插件
4. 在**隐身模式下仅使用简悦的方式** 

然后在试试 **融资约束、现金持有与过度投资**

**Reply by ilovepaper** [reply]
> 1. 关闭同步助手
> 2. 禁用出这个插件以外的全部插件
> 3. 今天我更新了下插件的功能，所以重新安装这个插件
> 4. 在**隐身模式下仅使用简悦的方式**
> 
> 然后在试试 **融资约束、现金持有与过度投资**

谢谢您的耐心！依旧是x.cnki.net/read/article域名，[链接参考](https://x.cnki.net/read/article/xmlonline?filename=JRYJ200907013&tablename=CJFDTOTAL&dbcode=CJFD&topic=&fileSourceType=1&taskId=&from=&groupid=&appId=CRSP_BASIC_PSMC&act=&ts=1647335817&customReading=#circle=on)
标题：融资约束、现金持有与过度投资。

我在另一台电脑上安装了简悦插件，知网插件外无任何其他插件，开隐私模式，重新安装该知网辅助插件后，测试结果如下：

不开启知网辅助插件的情况下，能顺利获取文本，但图片裂开；
![image](https://user-images.githubusercontent.com/57012991/158361591-7e8ffbeb-8984-40dd-b0fa-b6e3132caebe.png)

开启知网辅助插件正文和图片都无法获取。
![image](https://user-images.githubusercontent.com/57012991/158361505-8f7460fc-f71c-41dd-86c2-5cf0d2ae40b7.png)


我想本质上就是x.cnki.net/read/article域名下阅读和通过vpn然后进入html阅读网页格式是有差异的。连接vpn阅读论文不是大的问题，能够通过html阅读就已经很棒很方便了！非常非常感谢您的付出！（也很感谢前面其他提出这个需求的朋友）

ps：如果有其他朋友测试 x.cnki.net/read/article 域名下的知网能够顺利获取全文和图片，那就可能依旧是我这边的问题。

**Reply by Kenshin** [reply]
emmm，我大概了找了四五位用户帮忙测试，都是可以正常获取的，但区别是：

> 他们使用的 URL 是这样的结构 `https://kns.cnki.net/KXReader/Detail?invoice=xxxx` （跟我使用的测试是类似的 URL，但跟你的差别很大。

用你的邮件给我 `help#####simpread.pro` 邮箱里面（注意 `#` → `@`）并注明这个 Discussions ID：2103

我在邮件中告诉你一个办法。

**Reply by ilovepaper** [reply]
> emmm，我大概了找了四五位用户帮忙测试，都是可以正常获取的，但区别是：
> 
> > 他们使用的 URL 是这样的结构 `https://kns.cnki.net/KXReader/Detail?invoice=xxxx` （跟我使用的测试是类似的 URL，但跟你的差别很大。
> 
> 用你的邮件给我 `help#####simpread.pro` 邮箱里面（注意 `#` → `@`）并注明这个 Discussions ID：2103
> 
> 我在邮件中告诉你一个办法。

已发送邮件。谢谢！

**Comment by afenc** [comment]
https://jamanetwork.com/journals/jama/currentissue
JAMA系列文章output缺title
有空請適配

**Reply by Kenshin** [reply]
okay 收到~

**Comment by wangde-fu** [comment]
您好，感谢您的适配。
我在最近的使用中，偶然发现ieee期刊失去了适配，而在之前可以正常获取正文。现在，简阅只能智能获取到abstract部分。
[这是一篇openaccess的论文](https://ieeexplore.ieee.org/document/9687548)，如果您有空，希望能够抽空适配，谢谢。

**Reply by Kenshin** [reply]
okay 我瞧瞧是不是页面结构改变了，修复后会在这里通知你~

**Comment by Silix-byte** [comment]
https://pubmed.ncbi.nlm.nih.gov/
大佬，有空适配一下这个学术网站呗，谢谢了

**Reply by Kenshin** [reply]
okay 

但我需要一些具体的 URL （毕竟隔行如隔山 😂 我没法判断哪些是需要适配的~
